src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -611,6 +623,24 @@ namespace OpenRCT2::Ui::Windows
                         newListItem.date_formatted = Platform::FormatShortDate(newListItem.date_modified);
                         newListItem.time_formatted = Platform::FormatTime(newListItem.date_modified);
 
+                        // File size
+                        newListItem.fileSizeBytes = Platform::GetFileSize(newListItem.path.c_str());
+                        if (newListItem.fileSizeBytes > (1024 * 1024))
+                        {
+                            newListItem.fileSizeFormatted = newListItem.fileSizeBytes / (1024 * 1024);
+                            newListItem.fileSizeUnit = STR_SIZE_MEGABYTE;
+                        }
+                        else if (newListItem.fileSizeBytes > 1024)
+                        {
+                            newListItem.fileSizeFormatted = newListItem.fileSizeBytes / 1024;
+                            newListItem.fileSizeUnit = STR_SIZE_KILOBYTE;
+                        }
+                        else
+                        {
+                            newListItem.fileSizeFormatted = newListItem.fileSizeBytes;
+                            newListItem.fileSizeUnit = STR_SIZE_BYTE;
+                        }
```

Comment:
Maybe we want to do something like:

```c++
constexpr int kKiloByte = 1024;
constexpr int kMegaByte = 1024*1024;
```

Then use those instead?

Reply:
Nitpick: the strings display the sizes in kibibytes and mebibytes - as they should - but the code refers to them as kilobytes/megabytes. It would be cleaner if the code used the same terminology.